src/RCDEF.CPP

```diff
@@ -621,7 +621,7 @@ int CDEC main( SI argc, char * argv[] )
 		cFilesDir = argv[10];
 		if (!filesys::exists(filesys::path(cFilesDir)))
 		{
-			printf("\n'%s': Path not found.\n", incdir);
+			printf("\n'%s': C++ file output directory path not found.\n", incdir);
```

Comment:
@galanCA there is an error on this line.

Reply:
Fix.